scripts/subctl.sh

```diff
@@ -31,10 +31,9 @@ checkout_project_branch
 pushd projects/submariner-operator
 dapper_in_dapper
 
-export DEFAULT_IMAGE_VERSION=${release["version"]}
-export VERSION=${release["version"]}
-[[ "$1" == "cross" ]] && make build-cross
-make bin/subctl
+makeflags="VERSION=${release['version']} DEFAULT_IMAGE_VERSION=${release['version']}"
+[[ "$1" == "cross" ]] && make build-cross MAKEFLAGS="$makeflags"
+make bin/subctl MAKEFLAGS="$makeflags"
```

Comment:
I know I’ve used `MAKEFLAGS` shenanigans in the past, but I’m now of the opinion that we should let Make do its own thing (using `MAKEFLAGS`, but as an internal detail). Does this work instead?

```suggestion
[[ "$1" == "cross" ]] && make build-cross $makeflags
make bin/subctl $makeflags
```

or more explicitly,

```suggestion
[[ "$1" == "cross" ]] && make build-cross VERSION="${release['version']}" DEFAULT_IMAGE_VERSION="${release['version']}"
make bin/subctl VERSION="${release['version']}" DEFAULT_IMAGE_VERSION="${release['version']}"
```

Reply:
Thanks for reviewing :)

I first tried
```bash
make bin/subctl "$makeflags"
```

but for some reason it was failing because of the quotes.

I updated it to be without quotes and added shellcheck disable comments (as one does)

Reply:
Your usual trick of using an array should work here too ;-).

Reply:
Exactly what I did :grinning: